mtcpclient/calculate_test.go

```diff
@@ -35,23 +36,27 @@ func TestCalculateStdDev(t *testing.T) {
 		},
 	}
 	for _, test := range stdDevScenariosChecks {
-		var gc GroupOfConnections = []tcpclient.Connection{}
+
+		var gc *GroupOfConnections
+		gc = newGroupOfConnections(0)
 
 		var sum int
 		for i, connectionDuration := range test.durationsInSecs {
-			gc = append(gc, tcpclient.NewConnection(i, tcpclient.ConnectionEstablished,
+
+			gc.connections = append(gc.connections, tcpclient.NewConnection(i, tcpclient.ConnectionEstablished,
 				time.Duration(connectionDuration)*time.Second))
 			sum += connectionDuration
 		}
 
 		mr := metricsCollectionStats{}
```

Comment:
should we use newMetricsCollectionStats, now that we have one?

Reply:
I haven't seen this 😓